core-api/src/main/java/com/optimizely/ab/Optimizely.java

```diff
@@ -1842,6 +1950,16 @@ public Builder withODPManager(ODPManager odpManager) {
             return this;
         }
 
+        public Builder withCmabClient(CmabClient cmabClient) {
```

Comment:
@FarhanAnjum-opti It looks like we need to build withCmabService(), so others like android-sdk can inject full-featured CmabService with client + cacheSize + cacheTTL.
We can provide Builder pattern to CmabService like OdpManager - 

`
CmabServiceBuilder()
.withCmabCacheSize()
.withCmabCachTimeoutInSecs()
.withClient()
.build()
`